galaxy.yml

```diff
@@ -0,0 +1,23 @@
+# See https://docs.ansible.com/ansible/latest/dev_guide/collections_galaxy_meta.html
+
+namespace: community
+name: hashi_vault
+version: 0.1.0
+readme: README.md
+authors:
+  - YOUR NAME (github.com/YOURGITHUB)
```

Comment:
I guess we can use authors from the plugin ?
https://github.com/ansible-collections/community.hashi_vault/blob/8248977742411ddf22223dad253755377f96102f/plugins/lookup/hashi_vault.py#L12-L13

Reply:
agreed. thanks!